### PR TITLE
Adding Sandbox project: Meshery

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

### CODE REVIEW

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :)

**Comment by DelusionalOptimist** [comment]
Hello @carolynvs :wave:.
Please take a look at this PR and https://github.com/cncf/contribute/pull/68. I think both of them are good to get in.

Thanks for https://contribute.cncf.io/contributors. It was the first place I arrived at back when I was a newcomer and it was very helpful for me. It would be awesome if newcomers could find about Meshery and SMP too from here. :smile:

**Comment by leecalcote** [comment]
@mrbobbytables @cncf/sig-contributor-strategy

File: content/contributors/projects/_index.md [hunk 1]
```diff
@@ -529,6 +529,18 @@ Sandbox Projects
 -	**License:** [Apache 2.0](https://choosealicense.com/licenses/apache-2.0/)
 -	**Legal Requirements:** [DCO](https://developercertificate.org/)
 
+### Meshery
+
+*"Meshery is the open source, service mesh management plane that enables the adoption, operation, and management of any service mesh and their workloads."* - [Meshery](https://meshery.io)
+
+-	**Project Repository:** https://github.com/meshery
+-	**Contributor Guide:** [docs.meshery.io/project/contributing](https://docs.meshery.io/project/contributing)
+-	**Chat:** Slack: `#tinkerbell` in [slack.layer5.io](https://slack.layer5.io)
```
Comment (mrbobbytables) [inline content/contributors/projects/_index.md:538]:
```suggestion
-	**Chat:** Slack: `#meshery` in [slack.layer5.io](https://slack.layer5.io)
```
Small change, I think it was meant to be the meshery channel^^:;;

Other than that, everything looks good 👍

**Review by idvoretskyi — APPROVED** [review]

**Comment by idvoretskyi** [comment]
@leecalcote can you please rebase, and we'll merge :)

**Pull request merged**